README.md

```diff
@@ -93,6 +93,24 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // The amount of time before a user's "Basic" authentication
                 // token expires.
                 "timeout_in_seconds": 3600
+            },
+
+            // Defines required OIDC related configuration
+            "oicd": {
+                // The ip address or hostname of the OIDC provider
+                "config_host": "oidc.example.com",
+
+                // The port the OIDC provider is listening on
+                "port": "8080",
```

Comment:
True. I think initially I left it as a string because it's also used internally used as a string, so I didn't have to convert it.

Will change for consistency.

Reply:
Yeah, I noticed why it was a string much later in the code review.

I think leaving it as a string is fine. The only concern with changing it to an integer is it could confuse admins.

Reply:
Ended up changing it. Can always change back if needed.